internal/app/go-filecoin/porcelain/miner.go

```diff
@@ -78,7 +77,8 @@ func MinerCreate(
 	var minerAddr address.Address
 	err = plumbing.MessageWait(ctx, smsgCid, func(blk *block.Block, smsg *types.SignedMessage, receipt *types.MessageReceipt) (err error) {
 		if receipt.ExitCode != uint8(0) {
-			return vmErrors.VMExitCodeToError(receipt.ExitCode, storagemarket.Errors)
+			// Dragons: do we want to have this back?
```

Comment:
Do you mean the mapping between exit codes and error values or something else?  If that's what you mean it seems nice to have but unimportant.